boardgame dice/FrequencyCount.swift

```diff
@@ -8,23 +8,37 @@
 
 import SwiftUI
 
-let LEFT_BRAILLE: [String] = ["\u{2800}", "\u{2840}", "\u{28C0}", "\u{28C4}", "\u{28E4}", "\u{28E6}", "\u{28F6}"]
-let RIGHT_BRAILLE: [String] = ["\u{2800}", "\u{2880}", "\u{28C0}", "\u{28E0}", "\u{28E4}", "\u{28F4}", "\u{28F6}"]
+let LEFT_BRAILLE: [String] = ["\u{2800}", "\u{2840}", "\u{28C0}", "\u{28C4}", "\u{28E4}", "\u{28E6}", "\u{28F6}", "\u{28F7}", "\u{28FF}"]
+let RIGHT_BRAILLE: [String] = ["\u{2800}", "\u{2880}", "\u{28C0}", "\u{28E0}", "\u{28E4}", "\u{28F4}", "\u{28F6}", "\u{28FE}", "\u{28FF}"]
 
 struct FrequencyCount: View {
     
     let roll: Int
     let count: Int
     
+    func dots(_ row: Int) -> Int {
+        if row < count / 8 {
+            return 8
+        }
+        if row > count / 8 {
+            return 0
+        }
+        return count % 8
+    }
+    
     var body: some View {
-        VStack {
-            Text(LEFT_BRAILLE[count]).scaledFont(name: "Georgia", size: 24)
+        let braile = roll > 7 ? LEFT_BRAILLE : RIGHT_BRAILLE
+        VStack(alignment: .center) {
+            ForEach((0...max(1,count/8)).reversed(), id: \.self) { row in
```

Comment:
It was hard to figure out what this is doing.

This is the bit updating the counters so they can count to infinity? So it's figuring out the number of full multiples of 8 and then the remainder as a partial one?

I wonder if it's clearer to express along the lines of (in Python for lack of understanding Swift):

```python
def make_count_charcter_indices(count: int) -> List[int]:
  indices = []
  while count > 8:
    indices.append(8)
    count -= 8
  indices.append(count)
  return indices
```

and then iterator over those indices to render it.